BUILD.md

```diff
@@ -37,9 +37,9 @@ To build a container image of `openshift-ansible` using standalone **Docker**:
 
 To build an openshift-ansible image using an **OpenShift** [build and image stream](https://docs.openshift.org/latest/architecture/core_concepts/builds_and_image_streams.html) the straightforward command would be:
 
-        oc new-build registry.centos.org/openshift/playbook2image~https://github.com/openshift/openshift-ansible
+        oc new-build openshift/origin-ansible~https://github.com/openshift/openshift-ansible
```

Comment:
I'm sure there's a way to invoke the docker builder but I don't particularly want to try and test it right now, so save that for later.